recipes/gateway.rb

```diff
@@ -39,21 +39,24 @@
       false
     end
 
-  password = node['cdap']['cdap_site']['router.ssl.keystore.password']
-  keypass =
-    if node['cdap']['cdap_site'].key?('router.ssl.keystore.keypassword')
-      node['cdap']['cdap_site']['router.ssl.keystore.keypassword']
-    else
-      node['cdap']['cdap_site']['router.ssl.keystore.password']
-    end
-  path = node['cdap']['cdap_site']['router.ssl.keystore.path']
-  common_name = node['cdap']['security']['ssl_common_name']
-  jks =
-    if node['cdap']['cdap_site'].key?('router.ssl.keystore.type') && node['cdap']['cdap_site']['router.ssl.keystore.type'] != 'JKS'
-      false
-    else
-      true
-    end
+  if ssl_enabled.to_s == 'true'
```

Comment:
This ``if`` within the resource, and the resource's guard here: https://github.com/caskdata/cdap_cookbook/blob/feature/cook-75/recipes/gateway.rb#L63, both check if ``ssl_enabled`` is true.  It shouldn't be needed in both places.  (And it's extra wierd that the ``jks`` variable used in the 2nd half of the guard is now conditionally defined.)  It's not related to the fix in ``attributes/security.rb`` to prevent writing out to cdap-security.xml